js/ui/viewSelector.js

```diff
@@ -876,20 +828,57 @@ const ViewSelector = new Lang.Class({
     },
 
     _showPage: function(page, doFadeAnimation) {
-        if (!Main.overview.visible)
-            return;
-
         if (page == this._activePage)
             return;
 
         let oldPage = this._activePage;
         this._activePage = page;
         this._pageChanged();
 
-        if (oldPage)
-            this._animateOut(oldPage, doFadeAnimation)
-        else
-            this._animateIn(null, doFadeAnimation);
+        if (oldPage && doFadeAnimation) {
+            // When fading to the apps page, tween the opacity of the
+            // clone instead, and set the apps page to full solid immediately
+            if (page == this._appsPage) {
+                page.opacity = 255;
+                this._overviewViewsClone.opacity = AppDisplay.EOS_INACTIVE_GRID_OPACITY;
```

Comment:
Do you need to set the saturation here as well?

Reply:
Oops! Yes, probably. I took the code from the old shell, but I agree we should set the initial value